tests/ffi/network/test_largepckts_icmp.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+#Here $timper is allowed percentage delay and $ploss is allowed packet loss
+timper=5
+ploss=0
+echo " "
+echo "Creating Containers*"
+echo " "
+podman run -d --name orderly stream9-stress tail -f /dev/null
```

Comment:
I think the tests should run in a different way
U assume that orderly should run with ASIL filesystem space while confusion should be in QM, 
Am i correct?